Source/gui/SNTNotificationManager.m

```diff
@@ -378,7 +381,7 @@ - (void)postBlockNotification:(SNTStoredEvent *)event
                                                   configState:configState
                                                         reply:replyBlock];
 
-  [self queueMessage:pendingMsg];
+  [self queueMessage:pendingMsg ignoringSilences:(configState.enableNotificationSilences != YES)];
```

Comment:
nit: consider changing this param name to `enableSilences` or something, having the logic flip confused when reading the code.

Reply:
Done